lib/output-styles-preprocessor.js

```diff
@@ -24,8 +25,17 @@ OutputStylesPreprocessor.prototype.toTree = function(inputNode, inputPath, outpu
 
   debug('concatenating module stylesheets: %o', concatOptions);
 
+  var parentOptions = options.registry.app.options || {};
+  var cssModulesOptions = parentOptions.cssModules || {};
+  var postprocessPlugins = cssModulesOptions.plugins && cssModulesOptions.plugins.postprocess;
```

Comment:
You can call [`this.owner.getPlugins()`](https://github.com/alexlafroscia/ember-css-modules/blob/master/index.js#L87-L89) to simplify this